test_examples/local_terraform_module_source_subdirs/live/production/subdir/resource/terragrunt.hcl

```diff
@@ -0,0 +1,11 @@
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "${find_in_parent_folders()}/../../..//modules/subdir/resource"
```

Comment:
Is this intended to have a double slash? @Chancebair

Reply:
Yes, the double slash tells terragrunt that this is a subdir so that when modules are imported you don't import the entire repository into the terragrunt cache folder: https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories

Reply:
My understanding has been that the `//` subdir is only used for remote data sources. Of the four examples on that site,

- `hashicorp/consul/aws//modules/consul-cluster` is on the remote registry
- `git::https://example.com/network.git//modules/vpc` uses git/https
- `https://example.com/network-module.zip//modules/vpc` uses https
- `s3::https://s3-eu-west-1.amazonaws.com/examplecorp-terraform-modules/network.zip//modules/vpc` uses S3

What is the reason for using the slashes locally?

Reply:
Not using the double slash causes: "Error: Unreadable module directory: Unable to evaluate directory symlink" it's used to designate the local module root directory.  I'll try removing it in this PR as an example